src/lib/viewers/RuleViewer.ts

```diff
@@ -38,14 +38,31 @@ export class RuleDetailDisplayer extends AbstractRuleDisplayer {
 			const rule = rules[i];
 			const header = getMessage(BundleName.RuleViewer, 'summary.detail.header', [i + 1, rule.getName()]);
 			const severity = rule.getSeverityLevel();
+
 			const body = {
-				engine: rule.getEngineName(),
 				severity: `${severity.valueOf()} (${SeverityLevel[severity]})`,
-				tags: rule.getTags().join(', '),
-				resources: rule.getResourceUrls().join(', '),
-				description: rule.getDescription()
+				engine: rule.getEngineName(),
 			};
-			const keys = ['severity', 'engine', 'tags', 'resources', 'description'];
+			const keys: string[] = ['severity', 'engine'];
+
+			if (rule.getTags().length > 0) {
+				body['tags'] = rule.getTags().join(', ');
+				keys.push('tags');
+			}
```

Comment:
I think there might be value in having a tag-less rule display with `tags: none` or something like that.

Reply:
You'll have to ask John and Marcelino if that's what they want. Last I heard, we just wanted to hide the fields if they were empty.